src/simulate.js

```diff
@@ -17,18 +17,27 @@ var simulate = function (opts) {
     var target = opts.target || 'default';
     var simulator = new Simulator(opts);
     var showBrowser = opts.showbrowser;
+    var chromiumPath = opts.chromiumpath;
 
     if (!showBrowser) {
         var noBrowserMessage = 'The argument `showbrowser` is set to false. Please load simulated application in browser manually if needed.';
         log.warning(noBrowserMessage);
     }
 
+    if(process.platform === 'win32' && (target === 'chromium' || target === 'Chromium') && !chromiumPath){
```

Comment:
You can just use like `target = target.toLowerCase()` and verify.

Reply:
I remember trying this at first, but it was not working for some reason at time, maybe I had altered the target var type while testing it before. I tryied it again and seems to work fine, thanks!